src/fsharp/FSharp.Core/prim-types.fsi

```diff
@@ -2740,6 +2746,37 @@ namespace Microsoft.FSharp.Core
         [<CompiledName("KeyValuePattern")>]
         val ( |KeyValue| ): keyValuePair:KeyValuePair<'Key,'Value> -> 'Key * 'Value
 
+        [<AutoOpen>]
+        module ArrayExtensions = 
+            type ``[,,]``<'T> with
+                /// <summary>Get the index corresponding to ^i.</summary>
+                /// <param name="dim">The dimension of the index.</param>
```

Comment:
While the param name should still be `rank`, the description should probably describe what `rank` means in the context of the multi-dimensional array.